main.go

```diff
@@ -42,6 +42,9 @@ var (
 	setupLog = ctrl.Log.WithName("setup")
 )
 
+// +kubebuilder:rbac:groups="",resources=secrets,resourceNames=instascale-ocm-secret,verbs=get
+// +kubebuilder:rbac:groups="",resources=nodes;configmaps,verbs=list;watch;get;create;update;delete;patch
```

Comment:
Could this be narrowed to the minimal set of required permissions?

My understanding is that would be `get` verb for ConfigMaps, and `get`, `list`, `update`, `patch` for Nodes (maybe `update` is not needed).

Reply:
Hey @astefanutti,
I have pushed changes addressing your other comments and a separation of the `get verb for ConfigMaps`. I also moved the ocmsecret to instascale-system :)

In relation to the other renamings we spoke of, I feel this could be done in a follow-up pr focusing on renaming and reducing rbacs as there may be further discussion on that?

Reply:
Great, that looks good. I agree with your suggestion to follow this up when needed.